src/App.js

```diff
@@ -1,25 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './component/Calculator';
 
 function App() {
```

Comment:
- [ ] All components should be class-based. Kindly refactor this component accordingly